packages/neos-ui-editors/src/Editors/Button/index.js

```diff
@@ -0,0 +1,157 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Icon, IconButton} from '@neos-project/react-ui-components';
+import style from './style.module.css';
+
+import {neos} from '@neos-project/neos-ui-decorators';
+import {$transform} from 'plow-js';
+import {connect} from 'react-redux';
+import {selectors} from '@neos-project/neos-ui-redux-store';
+
+const getDataLoaderOptionsForProps = props => ({
+    contextNodePath: props.focusedNodePath,
+    dataSourceIdentifier: props.options.dataSourceIdentifier,
+    dataSourceUri: props.options.dataSourceUri,
+    dataSourceAdditionalData: props.options.dataSourceAdditionalData,
+    dataSourceDisableCaching: Boolean(props.options.dataSourceDisableCaching)
+});
+
+@neos(globalRegistry => ({
+    i18nRegistry: globalRegistry.get('i18n'),
+    dataSourcesDataLoader: globalRegistry.get('dataLoaders').get('DataSources')
+}))
+@connect($transform({
+    focusedNodePath: selectors.CR.Nodes.focusedNodePathSelector
+}))
```

Comment:
```suggestion
@connect(state => ({
    focusedNodePath: selectors.CR.Nodes.focusedNodePathSelector(state)
}))
```

as a replacement for `plow-js`'s `$transform` function.